internetarchive/openlibrary
# Remove jQuery `validate` from codebase

Closes #9605.


Refactor. The time has come at last!! Following in the footsteps of the previously merged subtasks (see issue page) which replaced each use of the jQuery `validate` plugin with built-in HTML validation and/or custom JS checks, this PR removes the last traces of the plugin from the codebase.

### Technical

Three basic courses of action:
1. Remove the `/validate` module import in `index.js` (which activates on any form with a validate class, of which there are now none!)
2. Delete `validate.js`, which was previously used to set the custom rules for the plugin
3. Uninstall `jquery-validation` via `npm`

### Testing

1. Search the codebase for `validate` to confirm that no forms have the class any longer
2. Search the codebase for `jquery-validation` to confirm that no other files import it
3. Search the codebase for `validate` again to confirm that no other files import the now-gone `validate.js` file

### Screenshot


### Stakeholders

@cdrini
